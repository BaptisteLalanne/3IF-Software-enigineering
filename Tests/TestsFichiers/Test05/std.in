1
5.6
32.7
40
2019-01-01
2019-01-04
0