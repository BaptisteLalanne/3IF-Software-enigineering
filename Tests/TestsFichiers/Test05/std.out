Acces aux donnees...
Donnees ajoutees avec succes!
4 capteurs non fiables detectes

Bienvenue dans le Menu Principal !

---------------------------MENU---------------------------
Choississez parmi les propositions suivantes
1: Calculer moyenne de la qualite de l'air dans une region
0: Quitter
Votre choix : ----------------------------------------------------------

Veuillez-renseigner les informations suivantes:
Longitude : Latitude : Rayon (km) : Date de debut (AAAA-MM-JJ) : Date de fin (AAAA-MM-JJ) :
Aucune donnee dans la zone ou au cours de la periode selectionnee.

---------------------------MENU---------------------------
Choississez parmi les propositions suivantes
1: Calculer moyenne de la qualite de l'air dans une region
0: Quitter
Votre choix : ----------------------------------------------------------

A bientot !
